ddtrace/contrib/botocore/patch.py

```diff
@@ -94,6 +94,14 @@ class TraceInjectionDecodingError(Exception):
     pass
 
 
+def _encode_data(trace_data):
```

Comment:
this is smart. I never think about adding methods solely to make testing easier/possible.

Reply:
A hard-learned lesson =p